src/IdentityManager/WebApi/appsettings.Development.json

```diff
@@ -7,7 +7,8 @@
     }
   },
   "ConnectionStrings": {
-    "DefaultConnection": "Data Source=(LocalDb)\\MSSQLLocalDB;Initial Catalog=TravelyIdentityService;Integrated Security=SSPI;AttachDBFilename=|DataDirectory|\\TravelyIdentityService.mdf"
+    "DefaultConnection": "Data Source=PCH-TADEVOSS\\MSSQLSERVER2012;Initial Catalog=TravelyIdentityService;Integrated Security=SSPI;"
```

Comment:
Revert this file.
Create `appsettings.local.json` and use it for local connection strings. Don't change this one.